doc/modules/changes/20250619_ranpengli

```diff
@@ -0,0 +1,4 @@
+Added: An entropy stastictics to the multicomponent averaging calculation. 
+It calculated that for every time step, what is the averaged iteration required for solving the equilibrated temperature when using entropy model with multiple components 
```

Comment:
This postprocessor calculates for every time step the average number of iterations to equilibrate the temperature between the multiple components.